src/sandbox_functions.cpp

```diff
@@ -1163,7 +1163,7 @@ PackedStringArray Sandbox::get_functions() const {
 	PackedStringArray result;
 	// Get all unmangled public functions from the guest program.
 	// Exclude functions that belong to the C/C++ runtime, as well as compiler-generated functions.
-	for (auto &function : machine().memory.all_unmangled_function_symbols()) {
+	for (std::string_view &function : machine().memory.all_unmangled_function_symbols()) {
```

Comment:
std::string_view by reference is almost always a bug, I think we need to have a look at this. If they are string_views, iterating it by value would be preferrable

Reply:
```
src/sandbox.cpp:569:4: error: 'this' argument to member function 'set' has type 'const SandboxProperty', but function is not marked const
                        prop.set(*this, value);
                        ^~~~
src/vmproperty.h:34:7: note: 'set' declared here
        void set(Sandbox &sandbox, const Variant &value);
             ^
1 error generated.
```

Reply:
Can you put a commit on top mentioning how you'll fix it?